### PR TITLE
Fix incorrect return when looking for a menu

### LINKED ISSUE #26: Search for a menu given a restaurant
- [X] Get menu route
- [X] Get menu model

### PR DESCRIPTION
**Pull request by WinsonChen114**

### CODE REVIEW

File: models/restaurant.js [hunk 1]
```diff
@@ -47,23 +53,25 @@ class Restaurant {
         //console.log("location:", location, city, postal_code, loc, restaurantName, OM_API_KEY)
         const result = await axios.get(`https://openmenu.com/api/v2/location.php?key=${OM_API_KEY}&country=us&${location}=${loc}&s=${restaurantName}`)
             .catch((err) => {
-                //console.log(err)
+                console.log(err)
```
Comment (ericloufoua) [inline models/restaurant.js]: REQUIRED: Remove console logs in your commits. Use Javascript Console Utils to help with making and removing them.

File: models/restaurant.js [hunk 2]
```diff
@@ -47,23 +53,25 @@ class Restaurant {
...
         return result.data.response.result.restaurants[0].id
     }
 
+    //Calls the API and returns details about a restaurant
     static async apiRestaurantInfo(id) {
         //console.log("in apiRestaurantInfo: id:", id)
         const result = await axios.get(`https://openmenu.com/api/v2/restaurant.php?key=${OM_API_KEY}&id=${id}`)
             .catch((err) => {
-                //console.log(err)
+                console.log(err)
```
Comment (ericloufoua) [inline models/restaurant.js]: REQUIRED: Remove Remove console logs in your commits.

**Review by ericloufoua — CHANGES_REQUESTED** [review]

File: models/restaurant.js (hunk 1 above)
Comment (WinsonChen114) [reply models/restaurant.js]: Removed

File: models/restaurant.js (hunk 2 above)
Comment (WinsonChen114) [reply models/restaurant.js]: Removed

**Pull request merged**